### ISSUE TITLE
Repeater cannot init itself on unsaved Static Page

### ISSUE THREAD

**Issue by Rike-cz** [octobercms/october#4158]
- OctoberCMS Build: 447
- PHP Version: 7.2

### Description:
When new Static Page is **not created**, Layout Repeater throws exception: `"A widget with class name 'formPaged41d8cd98f00b204e9800998ecf8427e5c6fee4869409ViewBagSlider' has not been bound to the controller" on line 554 of C:\......\fest\modules\backend\classes\Controller.php`.

After creating page this issue disappears.

### Steps To Reproduce:
![repeater-bug](https://user-images.githubusercontent.com/4247582/53243554-89740b00-36a8-11e9-8869-20a729e4d67e.gif)

**Comment by LukeTowers** [comment]
@Rike-cz this is a RainLab.Pages issue, not an October issue. Also, I think there's a related issue where it doesn't initialize when a page has had its layout switched but it hasn't been saved / reloaded.

**Comment by SaifurRahmanMohsin** [comment]
@LukeTowers This doesn't look like this is a plugin-specific issue. I can see the same issue in [one of my plugins](https://github.com/SaifurRahmanMohsin/oc-txt-plugin) which was working fine for the past 2 years.

**Comment by LukeTowers** [comment]
What do you mean you can see this issue in your plugin? This error only appears on the StaticPages pages which are only included by the RainLab.Pages plugin.

**Comment by SaifurRahmanMohsin** [comment]
I mean that I see the same exact error message on my plugin i.e. the repeater throws “A widget class name ‘formHumanFields’ has not been bound to the controller”. I tried creating a new repeater by defining it in the form with a single text field and I got the same error. So no idea what’s going on here. My plugin’s been there for 2 years and this broke recently so I’m thinking it might have to do with changes in OctoberCMS since it occured after I updated it. I’ll have to analyse it to confirm.
